demos/golang/grpc_benchmark/download_and_build_grpc_benchmark.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+export GOPATH=$PWD
+out_dir=$PWD/bin
+occlum-go get -u google.golang.org/grpc
+cd src/google.golang.org/grpc/
+git reset --hard 0f7e218c2cf49c7b0ca8247711b0daed2a07e79a
```

Comment:
Why do we need to use this special commit?

Reply:
I will remove it.